ReactSkia/external/BUILD.gn

```diff
@@ -2,6 +2,7 @@
 
 source_set("ReactSkiaExternal") {
   deps = [
+          "//ReactSkia/external/react-native-keyevent/skia:RnRNKeyEvent",
```

Comment:
my understanding is these entries will be added by codegen or any other in future. So we do not add these thirdparty package entries